packages/generator/src/WireOntologyDefinition.ts

```diff
@@ -15,7 +15,19 @@
  */
 
 import type * as gateway from "@osdk/gateway";
+import type { ObjectTypeV2 } from "@osdk/gateway/types";
 
 export interface WireOntologyDefinition
   extends gateway.components.OntologyFullMetadata
+{
+}
+
+export interface __UNSABLE_InterfaceType
```

Comment:
```suggestion
export interface __UNSTABLE_InterfaceType
```

^ this but do a global find and replace everywhere that's marked `UNSABLE`